index.html

```diff
@@ -6,10 +6,45 @@
     <link rel="stylesheet" href="css/index.css" />
     <title>Abdullahi Js Calculator</title>
   </head>
-  <body>
-    <main role="main">
-      <h1>Hello, World</h1>
+  <body class="theme1">
+    <main role="main" class="main-container d-flex">
+      <header class="header d-flex">
+        <h1 class="header-text">calc</h1>
+        <h2 class="header-txt">THEME</h2>
+        <div class="theme-container">
+          <div class="theme-input">
+            <div class="dot"></div>
+            <label class="theme-1-label" for="theme-1">1</label>
+
+            <label class="theme-2-label" for="theme-2">2</label>
+
+            <label class="theme-3-label" for="theme-3">3</label>
+          </div>
+        </div>
+      </header>
+      <div class="screen d-flex">
+        <div class="screen-input">399,981</div>
+      </div>
+      <div class="keypad d-grid">
+        <button class="key" value="7">7</button>
+        <button class="key" value="8">8</button>
+        <button class="key" value="9">9</button>
+        <button class="del" id="del">DEL</button>
+        <button class="key" value="4">4</button>
+        <button class="key" value="5">5</button>
+        <button class="key" value="6">6</button>
+        <button class="operator" value="+">+</button>
+        <button class="key" value="1">1</button>
+        <button class="key" value="2">2</button>
+        <button class="key" value="3">3</button>
+        <button class="operator" value="-">-</button>
+        <button class="key" value=".">.</button>
+        <button class="key" value="0">0</button>
+        <button class="operator" value="/">/</button>
+        <button class="operator" value="x">x</button>
+        <button class="reset" value="reset" id="reset-key">RESET</button>
+        <button class="equals-key equal" value="=" id="equals-key">=</button>
+      </div>
```

Comment:
Nice work with the button arrangements.

1. Please all buttons should have a value attribute, it would be used in your js
2. All buttons should have a type attribute `button`
3. All buttons should have a general class `key`, then an additional class for special cases like you are already doing
4. Let's avoid the use of `id` for these buttons, the value should be enough for all the features needed.